### PR TITLE
Remove script-like configuration from `cyhy-core` package files

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request removes any shebangs and executable bits remaining for files that are in the `cyhy_core` package.

> [!NOTE]
>The contents of the `test/` directory have been left as-is as they are not included when installing the library with `pip`.

## 💭 Motivation and context ##

Files in a Python package should not be configured like Python scripts would be with things like shebangs and executable bits.

## 🧪 Testing ##

I saw no issues deploying this in my testing environment and this change should not have any side-effects in general.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).

### CODE REVIEW

**Review by dav3r — APPROVED** [review]

File: cyhy/core/common.py
Comment (jsf9k) [inline cyhy/core/common.py:1]: Why does this empty file exist?

**Review by jsf9k — APPROVED** [review]
I approve, with one question.

File: cyhy/core/common.py
Comment (mcdonnnj) [reply cyhy/core/common.py:1]: This file had the executable bit removed but there are no changes to its contents (it is not an empty file).

Comment (jsf9k) [reply cyhy/core/common.py:1]: Ah, OK.  In the "Files changed" tab it lists the contents as "Empty file" which misled me.

_IGNORE ME!!!_

**Pull request merged**